### ISSUE TITLE
Is Gitmoji a necessary dependency?

### ISSUE THREAD

**Issue by 2gotgrossman** [frinyvonnick/node-html-to-image#24]
Hi @frinyvonnick, awesome package you've got here! I'm really excited to use it in production code, however I have a project running on Node v8.10.0 and the only blocker for installing this package is that `gitmoji-changelog` requires Node 10 or greater

```
$ yarn add-node-html-to-image
yarn run v1.10.1
error Command "add-node-html-to-image" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
$ yarn add node-html-to-image
yarn add v1.10.1
info No lockfile found.
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
[1/4] 🔍  Resolving packages...
warning node-html-to-image > gitmoji-changelog > immutadot > babel-runtime > core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
warning node-html-to-image > gitmoji-changelog > libnpm > npm-lifecycle > node-gyp > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
[2/4] 🚚  Fetching packages...
error gitmoji-changelog@2.1.0: The engine "node" is incompatible with this module. Expected version ">=10". Got "8.10.0"
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

**Comment by frinyvonnick** [comment]
Hi @2gotgrossman,

Thank you for reporting this issue! This is a mistake, it should be listed in devDependencies. Would you like to open a pull request to fix this?

**Comment by 2gotgrossman** [comment]
Done :) https://github.com/frinyvonnick/node-html-to-image/pull/25

**Comment by 2gotgrossman** [comment]
@frinyvonnick thanks for the quick response! I don't know the Publish to Yarn/NPM process. Is that something I can do after I merge this PR or is that something you have to do?

**Comment by frinyvonnick** [comment]
Thank you for opening a pull request. This is something I have to do. I'll do it as soon as possible 👍

**Comment by 2gotgrossman** [comment]
Thank you tons, @frinyvonnick. Do you know when you can merge the changes and publish them? :D

**Comment by frinyvonnick** [comment]
This has been merged in [2.1.0](https://github.com/frinyvonnick/node-html-to-image/releases/tag/v2.1.0) 🚀
